web/src/recent_view_ui.ts

```diff
@@ -845,26 +972,28 @@ function get_recent_view_filters_params() {
     };
 }
 
-function setup_dropdown_filters_widget() {
+function setup_dropdown_filters_widget(): void {
     filters_dropdown_widget = new dropdown_widget.DropdownWidget({
         ...views_util.COMMON_DROPDOWN_WIDGET_PARAMS,
         widget_name: "recent-view-filter",
         item_click_callback: filter_click_handler,
         $events_container: $("#recent_view_filter_buttons"),
+        // TODO: handle this `any`
```

Comment:
I asked about this on CZO https://chat.zulip.org/#narrow/stream/6-frontend/topic/typescript.20migration/near/1797854